doc/man/README.md

```diff
@@ -0,0 +1,96 @@
+LTTng-UST man pages
+===================
+
+This directory contains the sources of the LTTng-UST man pages.
+
+LTTng-UST man pages are written in
+[AsciiDoc](http://www.methods.co.nz/asciidoc/), and then converted to
+DocBook (XML) using the `asciidoc` command, and finally to troff using
+the appropriate DocBook XSL stylesheet (using the `xmlto` command).
+
+
+Custom XSL stylesheets
+----------------------
+
+There are a few custom XSL stylesheets applied for customizing the
+generated man pages in the `xsl` directory.
+
+
+Macros
+------
+
+AsciiDoc is configured with `asciidoc.conf` which contains a few
+macro definitions used everywhere in the man page sources.
+
+
+### `man`
+
+The `man` macro is used to specify a reference to another man page.
+Using the provided `asciidoc.conf` configuration file, the man page
+name is rendered in bold and the section is normal.
+
+Usage example: `man:lttng-enable-channel(1)`, `man:dlopen(3)`
+
+
+### `option`
+
+The option macro is used to write a command-line option which is
+**defined in the same man page**.
+
+Usage example: `option:--output`, `option:--verbose`
+
+
+### `nloption`
+
+Command-line option generating no link. This is used when writing
+about an option which is **not defined in the same man page**.
+
+Usage example: `nloption:-finstrument-functions`
+
+
+### `not`
+
+The `:not:` macro is used to emphasize on _not_.
+
+
+Includes
+--------
+
+  * `common-footer.txt`: common footer for all man pages.
+  * `common-footer-author.txt`: common footer plus the common author
+    of the LTTng-UST project. Only use this for man pages which describe
+    a command/library/function written by the main author of LTTng-UST.
+  * `log-levels.txt`: definition list of LTTng-UST log levels.
+  * `tracef-tracelog-limitations.txt`: limitations that apply to both
+    the `tracef(3)` and `tracelog(3)` man pages.
+
+
+Convention
+----------
+
+Please follow those rules when updating the current man pages or writing
+new ones:
+
+  * Always use macros when possible (man page references, command-line
+    options, _not_, etc.).
+  * Use callouts with the `term` role for command-line examples.
+  * Use a verse, possibly with the `term` role, in the synopsis section.
+  * Always refer to _long_ options in the text.
+  * Use the `option:--option='PARAM'` format (with `=`) when providing a
+    parameter to long options.
+  * Write _user space_, not _userspace_ nor _user-space_. (neither _user
+    land_).
+  * Write _file system_, not _filesystem_.
+  * Write _use case_, not _use-case_ nor _usecase_.
```

Comment:
there are valid cases for using "use-case" in English. It marks the association between use and case in a sentence where it would be other unclear what "use" should be associated with (previous word or following word).